Slambook/index.js

```diff
@@ -99,9 +78,47 @@ form.addEventListener("submit", (e) => {
```

Comment:
Consider adding some spacing or visual separation between the newly created elements for better readability.